engine/src/flutter/impeller/tessellator/tessellator.cc

```diff
@@ -576,9 +584,17 @@ ArcVertexGenerator Tessellator::StrokedArc(const Matrix& view_transform,
       ComputeQuadrantDivisions(view_transform.GetMaxBasisLengthXY() *
                                (arc.GetOvalSize().MaxDimension() + half_width));
 
-  return ArcVertexGenerator(arc.ComputeIterations(divisions),
-                            GetTrigsForDivisions(divisions),
-                            arc.GetOvalBounds(), half_width, cap);
+  std::unique_ptr<Trigs> round_cap_trigs;
+  if (cap == Cap::kRound) {
+    size_t round_cap_divisions = ComputeQuadrantDivisions(
+        view_transform.GetMaxBasisLengthXY() * half_width);
+    round_cap_trigs =
```

Comment:
There is also a GetTrigsForDeviceRadius that does the `ComputeQuadrantDivisions` for you.